clang/test/CodeGen/aie/looppragma.c

```diff
@@ -0,0 +1,51 @@
+// RUN: %clang --target=aie2 -S -emit-llvm %s -o - | FileCheck %s
```

Comment:
I think we can consider this as a target-independent test.

Reply:
true.